cle/backends/xbe.py

```diff
@@ -0,0 +1,150 @@
+from . import Backend, register_backend
+from ..errors import CLEError
+from ..patched_stream import PatchedStream
+from .region import Segment, Section, Region
+import archinfo
+from xbe import Xbe
+
+import logging
```

Comment:
Nitpick: Imports should be ordered like this: Python packages, third-party packages, angr packages, internal relative modules.

So the beginning of this file should be something like this:

```
import logging

from xbe import Xbe

import archinfo

from ..patched_stream import PatchedStream
from ..errors import CLEError
from . import Backend, register_backend
from .region import Segment, Section, Region
```

Reply:
Will fix. This was copy-pasted + extended from the blob backend, so I just kept what was there. I'll change it here before merge and we can go back and refactor blob (and possibly other modules) imports.

Reply:
Done